containers/ecr-viewer/src/app/admin/user/UserForm.tsx

```diff
@@ -0,0 +1,265 @@
+      formValid={valid}
+      submitAction={async () => {
+        const res = await submitAction(
+          email.trim(),
+          userType,
+          selectedPrograms,
+        );
+        if (!res.error)
+          createToast(`${email.trim()} successfully saved`, "success");
+        return res;
+      }}
+      successRoute="/admin/user"
+    >
+      <EmailFieldSet email={email} setEmail={setEmail} />
+      <UserTypeFieldSet userType={userType} setUserType={setUserType} />
+      <ProgramFieldSet
+        programs={programs}
+        setPrograms={setPrograms}
+        numProgramsSelected={numProgramsSelected}
+      />
+    </FormPageContent>
+  );
+};
+
+const EmailFieldSet = ({
+  email,
+  setEmail,
+}: {
+  email: string;
+  setEmail: (n: string) => void;
+}) => {
+  return (
+    <FieldSet legend="Email">
+      <span>
+        Add the new user by their login email (<RequiredMarker />)
+      </span>
+      <label className="usa-label">
+        EMAIL
+        <RequiredMarker />
+        <TextInput
+          type="email"
+          required={true}
+          id="email"
+          name="email"
+          value={email}
+          onChange={(e) => setEmail(e.target.value)}
+        />
+      </label>
+    </FieldSet>
+  );
+};
+
+const UserTypeFieldSet = ({
+  userType,
+  setUserType,
+}: {
+  userType: UserType;
+  setUserType: (n: UserType) => void;
+}) => {
+  return (
+    <FieldSet legend="Select user type">
+      <span>
+        User type will apply to all users added. These can be edited later (
+        <RequiredMarker />)
+      </span>
+      <label className="usa-label">
+        USER TYPE
+        <RequiredMarker />
+        {["admin", "standard"].map((option) => (
+          <Radio
+            key={option}
+            label={toTitleCase(option)}
+            type="radio"
+            required={true}
+            name="userType"
+            id={`userType-${option}`}
+            value={option}
+            checked={userType === option}
+            onChange={(e) => setUserType(e.target.value as UserType)}
+          />
+        ))}
+      </label>
+    </FieldSet>
+  );
+};
+
+const ProgramFieldSet = ({
+  programs,
+  setPrograms,
+  numProgramsSelected,
+}: {
+  programs: FormProgram[];
+  setPrograms: (c: FormProgram[]) => void;
+  numProgramsSelected: number;
+}) => {
+  const [expandedPrograms, setExpandedPrograms] = useState<
+    Record<string, boolean>
+  >(valsToBoolean(programs, true));
+
+  const accordionItems: AccordionItem[] = programs.map((program) => {
+    const name = program.name;
+    const conditions = program.conditions;
+    const numConditions = conditions.length;
+
+    return {
+      title: (
+        <div className="display-flex flex-justify flex-align-center">
+          <span>{name}</span>
+          <span>
+            {numConditions} condition{makePlural(numConditions)}
+          </span>
+        </div>
+      ),
+      content: conditions.map((c) => c.condition_name).join("; "),
```

Comment:
@ashton-skylight in the user details panel we join with `,` - what should we standardize on?

Reply:
right now we use `,` between each in a paragraph on the user details panel, but in this PR we use `; ` between each in a paragraph. A list is also an option, that's what we do in the conditions field in the program area details panel, but that one isn't an accordion 🙃